modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-type-fieldset/src/main/resources/META-INF/resources/fieldset_field.js

```diff
@@ -22,19 +34,168 @@ AUI.add(
 
 				prototype: {
 
+					initializer: function() {
+						var instance = this;
+
+						if (instance.get('repeatedIndex') > 0) {
+							instance.set('showLabel', false);
```

Comment:
This will trigger a call to `.render()`. We shouldn't be calling render on the initializer phase.  Better to move this logic to the render phase.